pkg/registry/core/node/strategy.go

```diff
@@ -125,9 +125,7 @@ func nodeConfigSourceInUse(node *api.Node) bool {
 func (nodeStrategy) Validate(ctx context.Context, obj runtime.Object) field.ErrorList {
 	node := obj.(*api.Node)
 	opts := validation.NodeValidationOptions{
-		// This ensures new nodes have no more than one hugepages resource
-		// TODO: set to false in 1.19; 1.18 servers tolerate multiple hugepages resources on update
-		ValidateSingleHugePageResource: true,
+		ValidateSingleHugePageResource: false,
```

Comment:
if this is unused everywhere in 1.19 now, remove the `ValidateSingleHugePageResource` option from the struct completely, and remove the ValidateNodeSingleHugePageResources method